homeassistant/components/renault/manifest.json

```
@@ -8,5 +8,6 @@
   "iot_class": "cloud_polling",
   "loggers": ["renault_api"],
   "quality_scale": "platinum",
-  "requirements": ["renault-api==0.1.12"]
+  "requirements": ["renault-api==0.1.12"],
+  "version": "0.1"
```

This is only for custom components - please remove version.
```suggestion
  "requirements": ["renault-api==0.1.12"]
```